components/BuildPage/NewToOlas.jsx

```diff
@@ -88,7 +88,7 @@ export const NewToOlas = () => (
           className="whitespace-normal max-sm:max-w-auto text-center max-sm:h-auto max-sm:py-2"
         >
           <a href={'https://www.valory.xyz/academy'}>
-            Explore Valory structured program
+            Explore Valory structured programs
```

Comment:
doesn't make sense.. use either one of the two:
```suggestion
            Explore Valory-structured programs
```
OR
```suggestion
            Explore Valory's structured programs
```